src/pages/theming/dark-mode.md

```diff
@@ -0,0 +1,267 @@
+---
+initialTab: 'preview'
+inlineHtmlPreviews: true
+previousText: 'Themes'
+previousUrl: '/docs/theming/themes'
+nextText: 'Advanced Theming'
+nextUrl: '/docs/theming/advanced'
+---
+
+# Dark Mode
+
+Ionic makes it easy to change the themes of your app, including supporting dark color schemes. With growing support for dark mode in native apps, developers are now looking to add it to their apps to support user preferences.
+
+## Using Media Queries
+
+The first way to enable dark mode is by using the CSS media query for the user's preferred color scheme. This media query will hook into the system setting of the user's device and apply the theme if a dark mode is enabled.
+
+```css
+@media (prefers-color-scheme: dark) {
+  :root {
+    /* dark mode variables go here */
+  }
+}
+```
+
+Currently, the `prefers-color-scheme` media query has [limited browser support](https://caniuse.com/#feat=prefers-color-scheme), so users will not be able to benefit from having the dark mode applied using this media query in certain browsers. However, the dark mode can still be applied by using a [class fallback](#class-fallback).
```

Comment:
Maybe we should denote here we mean a css class, not a JS/TS class. Just a thought.